assets/js/modules/analytics-4/components/dashboard/ActivationBanner/SetupBanner.js

```diff
@@ -191,22 +194,25 @@ export default function SetupBanner( { onSubmitSuccess } ) {
 
 		const { error } = await submitChanges();
 
+		setIsSaving( false );
```

Comment:
This is moved up to avoid a "state update on unmounted component" error from being raised as the success function called below triggers this component to be unmounted by the parent when we switch to render the success component.